pkg/logs/client/tcp/connection_manager.go

```diff
@@ -43,6 +43,10 @@ func NewConnectionManager(endpoint config.Endpoint) *ConnectionManager {
 	}
 }
 
+type timeoutError struct{}
+
+func (timeoutError) Error() string { return "tls: Handshake timed out" }
```

Comment:
nits: we don't use single line method so much in the code base, I'd rather not start this or can you provide a reason why we should start doing this please.